src/measure_theory/interval_integral.lean

```diff
@@ -43,20 +43,20 @@ name.
 ### One-sided derivatives
 
 Theorem `integral_has_fderiv_within_at_of_tendsto_ae` states that `(u, v) ↦ ∫ x in u..v, f x` has a
-derivative `(δu, δv) ↦ δv • cb - δu • ca` within the set `s × t` at `(a, b)` provided that `f` tends
-to `ca` (resp., `cb`) almost surely at `la` (resp., `lb`), where possible values of `s`, `t`, and
-corresponding filters `la`, `lb` are given in the following table.
+derivative `(δu, δv) ↦ δv • cb - δu • ca` within the set `s × t` at `(a, b)` provided that `f`
+tends to `ca` (resp., `cb`) almost surely at `la` (resp., `lb`), where possible values of `s`, `t`,
+and corresponding filters `la`, `lb` are given in the following table.
 
-| `s`     | `la`         | `t`     | `lb`         |
-| ------- | ----         | ---     | ----         |
+| `s`     | `la`          | `t`     | `lb`          |
+| ------- | ----          | ---     | ----          |
 | `Iic a` | `𝓝[Iic a] a` | `Iic b` | `𝓝[Iic b] b` |
 | `Ici a` | `𝓝[Ioi a] a` | `Ici b` | `𝓝[Ioi b] b` |
-| `{a}`   | `⊥`          | `{b}`   | `⊥`          |
+| `{a}`   | `⊥`           | `{b}`   | `⊥`          |
 | `univ`  | `𝓝 a`        | `univ`  | `𝓝 b`        |
```

Comment:
This table was correct before - if you count the unicode codepoints, the columns align in the "before" but don't align in the "after". To really see this, try matching the regex `\|.{9}\|.{14}\|.{9}\|.{14}\|` against each line. It matches all the lines on the left, but only some of the lines after your changes.

Can you just revert all the changes to this docstring?